posthog/hogql/placeholders.py

```diff
@@ -2,31 +2,35 @@
 
 from posthog.hogql import ast
 from posthog.hogql.errors import QueryError
+from posthog.hogql.utils import is_simple_value, deserialize_hx_ast
 from posthog.hogql.visitor import CloningVisitor, TraversingVisitor
 
 
-def replace_placeholders(node: ast.Expr, placeholders: Optional[dict[str, ast.Expr]]) -> ast.Expr:
-    return ReplacePlaceholders(placeholders).visit(node)
-
-
-def find_placeholders(node: ast.Expr) -> list[str]:
-    finder = FindPlaceholders()
-    finder.visit(node)
-    return list(finder.found)
-
-
 class FindPlaceholders(TraversingVisitor):
     def __init__(self):
         super().__init__()
-        self.found: set[str] = set()
+        self.has_expr_placeholders = False
+        self.has_filters = False
+        self.field_strings: set[str] = set()
```

Comment:
`field_strings` isn't a super descriptive name for what goes in here - I'm still not 100% sure what this is. I presume it'll be a set of things like `variables.some_variable_name`? Can we either add a docstring here or update the name?